tests/tests_e3/anod/test_sandbox.py

```diff
@@ -433,3 +433,25 @@ def test_anodtest(git_specs_dir, e3fake_git_dir):
         conf_fd.write(git_repo_entry)
     p = e3.os.process.Run(command)
     assert 'svn vcs type not supported' in p.out
+
+
+def test_failure_status(git_specs_dir, e3fake_git_dir):
+    root_dir = os.getcwd()
+    sandbox_dir = os.path.join(root_dir, 'sbx')
+    plan_file = os.path.join(root_dir, 'e3fake_test.plan')
+    p = e3.os.process.Run(['e3-sandbox', '-v', 'create',
+                           '--spec-git-url', git_specs_dir,
+                           sandbox_dir])
+    assert p.status == 0
+
+    with open(plan_file, 'w') as plan_fd:
+        plan_fd.write("anod_test('e3fake')")
+    conf_dir = os.path.join(sandbox_dir, 'specs', 'conf.yaml')
+    git_repo_entry = E3Fake % 'no_url'
+    with open(conf_dir, 'w') as conf_fd:
+        conf_fd.write(git_repo_entry)
+    command = ['e3-sandbox', '-v', 'exec',
+               '--plan', plan_file,
+               sandbox_dir]
+    p = e3.os.process.Run(command)
+    assert p.out.count('status=  1') >= 6
```

Comment:
why a space after the = sign? why 6? I have to say that I don't understand this test.